### PR TITLE
Add highlights, questions, and teams

### PR DESCRIPTION
**Pull request by crkingsley**

### CODE REVIEW

File: QUESTIONS.md [hunk 1]
```diff
@@ -3,3 +3,11 @@
 What questions can you come up with?
 
 ## Cohort Questions
+
+- Can we have 10-10-10 challenges around using REACT (small CRUD exercise?)
```
Comment (Emeralds99) [inline QUESTIONS.md:18]: Think a 10-10-10 on this stuff, maybe out functions vs. components would be great!

**Review by Emeralds99 — APPROVED** [review]
🏃 💨

File: QUESTIONS.md [hunk 2]
```diff
@@ -3,3 +3,11 @@
...
+- What are some ways to better manage burnout and paralysis analysis?
+- Advice on how to maintain continuity of thought when working on a long-term project?
+- How to use npkill?
```
Comment (hollisjamison) [inline QUESTIONS.md:24]: Good question,i started it up and it took forever then I just closed it

File: QUESTIONS.md (hunk 1 above)
Comment (zward2) [inline QUESTIONS.md:18]: +1

File: QUESTIONS.md (hunk 2 above)
Comment (SBucharelli) [reply QUESTIONS.md:24]: JR said he was going to go over this, correct?

**Review by SBucharelli — APPROVED** [review]

**Pull request merged**